### PR TITLE
[fix][client] Added checks before using publicKey and cleanup of expired chats

### PR DESCRIPTION
**Pull request by Tizzz-555**
This is more like a temporary workaround not to crash, I would like to understand better what would be your desired behaviour.
Do you want the chats to be permanent?

### CODE REVIEW

File: client/src/views/Chat.vue [hunk 1]
```diff
@@ -196,18 +196,40 @@ export default {
...
+    const publicKey = this.$cookies.get(id);
+    if (!publicKey || typeof publicKey !== 'string') {
+        console.error('Invalid publicKey for user:', id);
+        this.toastmsg = "Error: Invalid chat session.";
+        this.toastType = "error";
+        setTimeout(() => {
+            this.toastmsg = "";
+            this.toastType = "";
+        }, 2000);
+        return;
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:209]: Lets connect to the user again, using the `connect` method. If the public key is still invalid just disable the chat by disabling the `MessageForm`, the use can view existing messages but not send a new. Show the message `user is inactive` in the chat window.

WDYT?

**Review by PradyumnaKrishna — COMMENTED** [review]
Reviewed! Sorry for the late review.

File: client/src/views/Chat.vue (hunk 1 above)
Comment (Tizzz-555) [reply client/src/views/Chat.vue]: Just pushed some new changes, let me know if this could be ok! 😉

**Review by PradyumnaKrishna — APPROVED** [review]
LGTM! thank you @Tizzz-555 for this fix.

**Pull request merged**